scripts/generate-catalog.py

```
@@ -3,22 +3,33 @@
 import os
 from pathlib import Path
-import json
 import time
 import urllib
 import tarfile
+import re
+from datetime import timedelta, datetime
 
 from jinja2 import Environment
 import git
 from jinja2 import Environment, FileSystemLoader, select_autoescape
```

Ruff (0.8.2)

Redefinition of unused `Environment` from line 12

Remove definition: `Environment`

(F811)